packages/design-system/src/components/web-components/preactement/globalStyles.ts

```diff
@@ -0,0 +1,84 @@
+const POLLING_INTERVAL = 10;
```

Comment:
Curious to know how this specific value was chosen, given the potential performance impacts of frequent polling.

Reply:
I wanted it short enough to not see flashes of _mis_-styled content, and I figured since I was only checking the 2-10 or so stylesheet elements in a document and not accessing all their children that it would be fairly inexpensive. This is not based on any empirical data, though. Do you have a specific number in mind?

Reply:
Everyone vote on their favorite polling interval? 😄

Reply:
Also I hadn't thought of monitoring the performance, but do you have ideas?

Reply:
Thanks for clarifying the polling scope; it’s more limited than I realized, which is good to know. Regarding performance impacts—granted I haven't thought about this all that deeply—I was thinking about using browser profiling tools against our example angular application as a possible way to assess performance. I'm not super experienced with profiling tools, but that's the first thing that came to mind.

Reply:
Okay, I successfully profiled it and found that our code that runs every 10ms executes on average in 50μs (50 millionths of a second)!